contrib/docker/Dockerfile

```diff
@@ -1,18 +1,19 @@
-FROM docker.io/golang:alpine as builder
+FROM golang:alpine as builder
```

Comment:
Removing the registry here doesn't simplify the Dockerfile, it creates ambiguity, since different distributions and tools might default to different registries.